src/components/asset-characteristics/asset-characteristics.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+import { mockAssetV1, render } from "@hackney/mtfh-test-utils";
+import { screen } from "@testing-library/react";
+
+import { locale } from "../../services";
+import { PropertySpecification } from "./asset-characteristics";
+
+describe("AssetCharacteristics", () => {
```

Comment:
Maybe you should add another test to not display the characteristics if the value is null